$NetBSD$

Assume the assets directory is at the same level. This is pkgsrc-specific.

--- dist/index.html.orig	2025-03-08 08:16:43.689562933 +0000
+++ dist/index.html
@@ -10,7 +10,7 @@ SPDX-License-Identifier: AGPL-3.0-or-lat
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>DMARCaroni app</title>
-  <link rel="stylesheet" href="../assets/bulma.min.css">
+  <link rel="stylesheet" href="./assets/bulma.min.css">
   <script src="./main.js"></script>
 </head>
 <body>
